app/models/user.rb

```diff
@@ -383,7 +386,7 @@ def saw_notification_id(notification_id)
 
   def read_first_notification?
     if (trust_level > TrustLevel[1] ||
-        created_at < TRACK_FIRST_NOTIFICATION_READ_DURATION.seconds.ago)
+        first_seen_at < TRACK_FIRST_NOTIFICATION_READ_DURATION.seconds.ago)
```

Comment:
@tgxworld Do you have any objections to using `first_seen_at` instead of `created_at`? It shouldn't make a difference for regular users, but it's the easiest way to make it work for previously staged users.